website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -100,10 +104,29 @@ spec:
       serviceAccountName: static-server
 ```
 
+### Kubernetes HTTP Health Probes Configuration
+Traffic redirection interferes with [Kubernetes HTTP health
+probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) since the
+probes expect that kubelet can directly reach the application container on the probe's endpoint, but that traffic will
+be redirected through the sidecar proxy, causing errors because kubelet itself is not encrypting that traffic using a
+mesh proxy. For this reason, Consul allows you to overwrite Kubernetes HTTP health probes to point to the proxy instead,
+to pass that traffic through via the proxy using the Helm value `connectInject.transparentProxy.defaultOverwriteProbes`
```

Comment:
This second part of the sentence feels like it's repeating what the first part is saying, so I think we can omit it.
```suggestion
using the Helm value `connectInject.transparentProxy.defaultOverwriteProbes`
```